packages/eui/src/components/drag_and_drop/draggable.tsx

```diff
@@ -42,6 +43,13 @@ export interface EuiDraggableProps
    * Whether the item is currently in a position to be removed
    */
   isRemovable?: boolean;
+  /**
+   * Whether the currently dragged item is cloned into a portal in the body. This settings will
+   * ensure that drag & drop still works as expected within stacking contexts (e.g. within EuiFlyout and EuiModal).
```

Comment:
```suggestion
   * ensure that drag & drop still works as expected within stacking contexts (e.g. within `EuiFlyout`, `EuiModal`, and `EuiPopover`).
```
We should likely deprecate the `<EuiPopover hasDragDrop />` prop at this point and tell consumers to use this new `usePortal` prop instead. Could we also update `EuiDataGrid`'s usage of `hasDragDrop` as well if possible? 👀

Reply:
Speaking of EuiDataGrid, we can now also change these toolbars to using `euiYScrollWithShadows()` again:

https://github.com/elastic/eui/blob/c60a91c69e25808ca7278766695e3ce6bf339347/packages/eui/src/components/datagrid/controls/column_selector.styles.ts#L25

https://github.com/elastic/eui/blob/c60a91c69e25808ca7278766695e3ce6bf339347/packages/eui/src/components/datagrid/controls/column_sorting.styles.ts#L39

I changed them to just `euiYScroll` in the Emotion conversion because the shadows mask was causing the aforementioned stacking context bug - but now this awesome prop lets us have both!

Reply:
Updated in this [commit](https://github.com/elastic/eui/pull/8048/commits/5fa119590f08821147d6a05461b748b436a77ecf)

Reply:
Perfection!! I've also added a deprecation schedule task item for `<EuiPopover hasDragDrop />` for December: https://github.com/elastic/eui/issues/1469